crates/nu-command/src/filters/get.rs

```diff
@@ -41,8 +41,13 @@ If multiple cell paths are given, this will produce a list of values."#
             )
             .rest("rest", SyntaxShape::CellPath, "Additional cell paths.")
             .switch(
-                "ignore-errors",
+                "optional",
                 "ignore missing data (make all cell path members optional)",
```

Comment:
maybe we should change this description to put the "make all cell path members optional" more in the forefront